guvnor-structure/guvnor-structure-api/src/main/java/org/guvnor/structure/repositories/PullRequest.java

```diff
@@ -0,0 +1,16 @@
+package org.guvnor.structure.repositories;
+
+public interface PullRequest {
+
+    long getId();
```

Comment:
@Salaboy I'll let @adrielparedes clarify, but the commit's are all in the source repository branch.